types/types.go

```diff
@@ -107,6 +107,17 @@ func (a Address) Bytes() []byte {
 	return a[:]
 }
 
+// ExistsIn check if address exists in address list
+func (a Address) ExistsIn(addresses []Address) bool {
+	for _, address := range addresses {
+		if a == address {
+			return true
+		}
+	}
+
+	return false
+}
+
```

Comment:
Why define this as a method of the `Address` type?
Why not just define it where you use it

Reply:
i am using this method across multiple modules. Also this feels like util method which we are going to need in the future for other things. And it is all related with the address struct

Reply:
@dbrajovic @Kourin1996 what do you think?

Reply:
Why not define a helper in the helper module, instead of defining it as a type method?

Reply:
You convinced me 😎 9de90082c113b2a3349464d1f1dfa52e6a487add